services/management/dbaas/xtradb_cluster_service_test.go

```diff
@@ -133,7 +133,7 @@ func TestXtraDBClusterService(t *testing.T) {
 		resp, err := s.ListXtraDBClusters(ctx, &dbaasv1beta1.ListXtraDBClustersRequest{KubernetesClusterName: kubernetesClusterNameTest})
 		assert.NoError(t, err)
 		require.NotNil(t, resp.Clusters[0])
-		assert.Equal(t, resp.Clusters[0].Name, "first.pxc.test.percona.com")
+		assert.Equal(t, resp.Clusters[0].Name, "first-pxc-test")
```

Comment:
Should we also check user and pass are in the response?

Reply:
`ListXtraDBClusters` - should not return creds.